src/test/yaml.completion.test.ts

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as chai from 'chai';
+import { getDocUri, activate } from './helper';
+
+const expect = chai.expect;
+
+describe('YAML DSL and endpoint uri completion tests backed by schema', () => {
+	const yamlSchemaUri: vscode.Uri = vscode.Uri.parse('https://raw.githubusercontent.com/lburgazzoli/apache-camel-k-runtime/github-358/camel-k-loader-yaml/camel-k-loader-yaml/src/generated/resources/yaml-schema.json');
```

Comment:
it wasn't available when the PR was written but now that it is on master, maybe better to point to https://github.com/apache/camel-k-runtime/blob/master/camel-k-loader-yaml/camel-k-loader-yaml/src/generated/resources/camel-yaml-dsl.json

that said, it migth be worthy to wait for the schemastore to be available and rely on it https://github.com/SchemaStore/schemastore/pull/1138 (or can be done in another iteration)

Reply:
it is now available on schemastore so it can be completely removed